src/CalculatorUITestFramework/StandardAoTCalculatorPage.cs

```diff
@@ -160,5 +160,43 @@ private void GrowWindowToShowInvertButton(int height)
                 GrowWindowToShowInvertButton(width + 100);
             }
         }
+        /// <summary>
+        /// If the Invert button is not displayed, resize the window
+        /// Two attempts are made, the the button is not found a "not found" exception is thrown
+        /// </summary>
+        public void ResizeAoTWindowToDiplayInvertButton()
+        {
+            Point newWindowPostion = new Point(8, 8);
+            WinAppDriver.Instance.CalculatorSession.Manage().Window.Position = newWindowPostion;
```

Comment:
you don't need multiple window position and source objects. Just reuse the first one.